firstvoices/backend/views/search/base_search_views.py

```diff
@@ -379,10 +425,44 @@ def list(self, request, **kwargs):
             from_=pagination_params["start"], size=pagination_params["page_size"]
         )
 
-        # Sort by score, then by custom sort order
-        search_query = search_query.sort(
-            "_score", {"custom_order": {"unmapped_type": "keyword"}}, "title.raw"
+        sort_direction = (
+            "desc"
+            if search_params["descending"] and search_params["sort"] is not None
```

Comment:
If you always return None for descending when the sort param is missing or invalid, you can skip this type of check on the sort param value